lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -2207,15 +2207,15 @@ func TestMakeRemapDotConfigEdgeHostRegexReplacementHTTP(t *testing.T) {
 
 	txtLines := strings.Split(txt, "\n")
 
-	if len(txtLines) != 2 {
+	if len(txtLines) != 3 {
 		t.Fatalf("expected 1 remap from HTTP DS, actual: '%v' count %v", txt, len(txtLines))
```

Comment:
> "expected <del>1</del><ins>3</ins> remap<ins>s</ins> from..."

Reply:
There's only 1 remap, the other 2 lines are a comment and a blank